src/couch/priv/stats_descriptions.cfg

```diff
@@ -418,10 +422,47 @@
     {type, counter},
     {desc, <<"number of other requests">>}
 ]}.
+{[couchdb, query_server, js_filter], [
+    {type, counter},
+    {desc, <<"number of JS filter invocations">>}
+]}.
```

Comment:
There is already a `.query_server.calls.ddoc_filter` counter

Reply:
Ahhh good catch, I didn't see those were hidden in https://github.com/apache/couchdb/blob/main/src/couch/src/couch_os_process.erl#L257-L259

I've reworked this in `4db08c9f2` to use the existing metrics and only add a new metric `[couchdb, query_server, volume, ddoc_filter]` for counting the number of docs funneled through the filter. I also reworked the logic so that volume metric is captured in the same module and style as the `bump_time_stat` function above.